README.md

```diff
@@ -0,0 +1,40 @@
+# triSYCL
```

Comment:
I would prefer keeping the ReST format.

Reply:
Back-ported in https://github.com/keryell/triSYCL/commit/2fc09111c6325583ec70f8b14a3dcc8d005c8189